test-kotest5/src/main/kotlin/io/micronaut/test/extensions/kotest5/MicronautKotest5Extension.kt

```diff
@@ -27,14 +27,15 @@ import io.micronaut.test.extensions.kotest5.annotation.MicronautTest
 import kotlin.reflect.KClass
 import kotlin.reflect.full.primaryConstructor
 
-object MicronautKotest5Extension: TestListener, ConstructorExtension, TestCaseExtension {
+object MicronautKotest5Extension : TestListener, ConstructorExtension, TestCaseExtension {
 
     override suspend fun intercept(
         testCase: TestCase,
         execute: suspend (TestCase) -> TestResult
     ): TestResult {
-        val context = contexts[testCase.spec.javaClass.name]
-        return if(context != null && context.getSpecDefinition() == null) {
+//        val context = contexts[testCase.spec]
```

Comment:
Probably worth cleaning the commented code up?

Reply:
Yep thanks done.